app/_src/policies/meshpassthrough.md

```diff
@@ -0,0 +1,155 @@
+
+## TargetRef support matrix
+
+{% tabs targetRef useUrlFragment=false %}
+{% tab targetRef Sidecar %}
+| `targetRef`           | Allowed kinds         |
+| --------------------- | --------------------- |
+| `targetRef.kind`      | `Mesh`, `MeshSubset`  |
+{% endtab %}
+{% endtabs %}
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+## Configuration
+
+{% warning %}
+This policy doesn't work with sidecars not using [transparent-proxy](/docs/{{ page.version }}/networking/transparent-proxying/#what-is-transparent-proxying).
+{% endwarning %}
+
+The following describes the default configuration settings of the `MeshPassthrough` policy:
+
+- **`enabled`**: (Optional) If true, allows traffic to all external destinations.
+- **`appendMatch`**: List of destinations that sidecar should be able to reach. When `enabled` is `true` this list is not used. It works only when `enabled` is `false`.
+  - **`type`**: Defines what type of destination should be allowed. Either `Domain`, `IP` or `CIDR`
+  - **`value`**: Destination address based on the defined `type`.
+  - **`port`**: Port at which exterbal service is available. When not defined it caches all traffic to the address.
+  - **`protocol`**: Defines protocol of the external service.
+    - **`tcp`**: **Can't be used when `type` is `Domain` (at TCP level we are not able to disinguish domain, in this case it is going to hijack whole traffic on this port)**.
+    - **`tls`**: Should be used when TLS traffic is originated by the client application.
+    - **`http`**
+    - **`http2`**
+    - **`grpc`**
+  
+### Wildcard domains
+
+A `MeshPassthrough` policy allows you to create a match for a wildcard domain. You can match a subdomain and allow this traffic to go outside of the cluster.
+
+{% warning %}
+Currently, partial matches are not possible. For example, the domain `*ww.example.com` will be rejected.
+{% endwarning %}
+
+{% policy_yaml wildcard %}
+```yaml
+type: MeshPassthrough
+name: wildcard-passthrough
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+  default:
+    appendMatch:
+    - type: Domain
+      value: '*.cluster-1.kafka.aws.us-east-2.com'
+      protocol: tls
+      port: 443
+```
+{% endpolicy_yaml %}
+
+### Security
+
+It is advised that the `MeshOperator` is responsible for the `MeshPassthrough` policy. This policy can introduce traffic outside of the mesh or even the cluster, and the `MeshOperator` should be aware of this. If you want to secure access to `MeshPassthrough` to specific services, you must choose them manually. If you rely on tags in the top-level `targetRef` you might consider securing them by one of the:
+
+* Make sure that service owners can't freely set them (using something like [`kyverno`](https://kyverno.io/), [`OPA`](https://www.openpolicyagent.org/))
+* Accept the risk of being able to "impersonate" a passthrough label and rely on auditing.
```

Comment:
Would be great if we had this https://github.com/kumahq/kuma-website/issues/927 so we can link to personas. Let's split this into multiple lines so it's easier to track in git.

```suggestion
It is advised that the `MeshOperator` is responsible for managing the `MeshPassthrough` policy.
This policy can introduce traffic outside of the mesh or even the cluster, and the `MeshOperator` should be aware of this.
If you want to restrict access to `MeshPassthrough` to specific services, you must choose them manually.
If you rely on tags in the top-level `targetRef` you might consider securing them by using one of the following techniques:

* Make sure that service owners can't freely modify them (using something like [`kyverno`](https://kyverno.io/), [`OPA`](https://www.openpolicyagent.org/) or similar)
* Accept the risk of being able to "impersonate" a passthrough label and rely on auditing to figure out any violations.
```